sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -633,6 +656,21 @@ public partial class CustomNormalizer : Azure.Search.Documents.Indexes.Models.Le
         public System.Collections.Generic.IList<Azure.Search.Documents.Indexes.Models.CharFilterName> CharFilters { get { throw null; } }
         public System.Collections.Generic.IList<Azure.Search.Documents.Indexes.Models.TokenFilterName> TokenFilters { get { throw null; } }
     }
+    public partial class CustomVectorizer : Azure.Search.Documents.Indexes.Models.VectorSearchVectorizer
+    {
+        public CustomVectorizer(string name) : base (default(string)) { }
+        public Azure.Search.Documents.Indexes.Models.CustomVectorizerParameters CustomVectorizerParameters { get { throw null; } set { } }
+    }
+    public partial class CustomVectorizerParameters
+    {
+        public CustomVectorizerParameters() { }
+        public Azure.Search.Documents.Indexes.Models.SearchIndexerDataIdentity AuthIdentity { get { throw null; } set { } }
+        public string AuthResourceId { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, string> HttpHeaders { get { throw null; } }
+        public string HttpMethod { get { throw null; } set { } }
+        public System.TimeSpan? Timeout { get { throw null; } set { } }
+        public string Uri { get { throw null; } set { } }
```

Comment:
Why not a `Uri`?

Reply:
Note: if their swagger/TypeSpec doesn't specify the `format: uri` property, they should update their swagger. It'd probably be flagged as a breaking change, but should be approved fairly easily if it was supposed to be a URI the entire time. It's more of a hint.